ui/nuxeo-user-group-management/nuxeo-user-management.js

```diff
@@ -551,10 +551,7 @@ import './nuxeo-user-group-permissions-table.js';
     }
 
     _hasAdministrationPermissions(currentUser) {
-      return (
-        currentUser &&
-        (currentUser.isAdministrator || (this.isMember(currentUser, 'powerusers') && !this.user.isAdministrator))
-      );
+      return this.hasAdministrationPermissions(currentUser) && !this.user.isAdministrator;
```

Comment:
If `this.user.isAdministrator` then it will always return `false` and `currentUser` won't be able to proceed even if `currentUser.isAdministrator` is `true`

Reply:
unfortunately i didn't pay more attention,
thx